.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/onqtam/doctest.git
 [submodule "3rdparty/SafeInt"]
 	path = 3rdparty/SafeInt
-	url = https://github.com/dcleblanc/SafeInt
+	url = https://github.com/rsmmr/SafeInt.git
```

Comment:
@rsmmr, do you plan to upstream your patch? Also, using a private fork is not ideal as it makes pulling in upstream fixes impossible for anyone but you.